Source/Core/Core/PowerPC/Jit64/Jit_Integer.cpp

```diff
@@ -662,6 +662,152 @@ void Jit64::boolX(UGeckoInstruction inst)
     else if (inst.SUBOP10 == 284)  // eqvx
       gpr.SetImmediate32(a, ~(rs_offset ^ rb_offset));
   }
+  else if (gpr.IsImm(s) || gpr.IsImm(b))
+  {
+    auto [i, j] = gpr.IsImm(s) ? std::pair(s, b) : std::pair(b, s);
```

Comment:
Maybe mark these as locals as const where applicable so readers know that most of these won't change, and don't need to think about ones that are otherwise immutable.

Reply:
Makes sense!